internal/storage/image.go

```diff
@@ -420,6 +438,209 @@ func (svc *imageService) imageStatus(systemContext *types.SystemContext, unstabl
 	return &result, nil
 }
 
+func (svc *imageService) PullLocalManifestDigest(ctx context.Context, systemContext *types.SystemContext, userSpecifiedImage string) (string, error) {
```

Comment:
(Ignoring this because I can’t see a caller.)

Reply:
I agree, that one can be dropped.